apps/app_speech_utils.c

```diff
@@ -933,6 +933,11 @@ static int speech_background(struct ast_channel *chan, const char *data)
 		}
 	}
 
+	/* Copy to speech structure the results, even partial ones, if available */
```

Comment:
This behavior should be documented in the documentation for the dialplan application.